docs/UserGuide.md

```diff
@@ -541,6 +543,18 @@ corrupt data will be discarded by ConTech upon launch. Users are recommended to
 app instead of manually editing the save data files. For adding a large number of contacts at once, the `import` function
 can be used.
 
+**Q**: Does ConTech support duplicate prevention?
+
+**A**: Yes, ConTech has built-in duplicate protection for all commands except `import` currently. This will be added in
+a future revision with more granular controls to allow users to easily handle multiple duplicate contacts when importing.
+
+Current implementation ideas include:
+- import all with/without duplicates (use flag to specify)
+- manually check every single duplicate (inefficient, but can be an option with flag for users who want to check every duplicate)
+- allow Windows-style duplicate handling (Ignore, Ignore all, Cancel)
+- summarise every single duplicate and give user a one time confirmation to add/discard (no granularity)
```

Comment:
Good work listing out the potential ideas for implementing the duplicate protection, but I was thinking would it be better to leave this out and place it in Developer Guide, since it sounds a little more technical and more suitable for Developer Guide. It might not be something users are looking out for.